test_unstructured_ingest/python/test-databricks-volumes.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+import json
+
+import click
+from databricks.sdk import WorkspaceClient
+
+
+@click.group()
+def cli():
+    pass
+
+
+def _get_volume_path(catalog: str, volume: str, volume_path: str):
+    return f"/Volumes/{catalog}/default/{volume}/{volume_path}"
+
+
+@cli.command()
+@click.option("--host", type=str, required=True)
+@click.option("--username", type=str, required=True)
+@click.option("--password", type=str, required=True)
+@click.option("--catalog", type=str, required=True)
+@click.option("--volume", type=str, required=True)
+@click.option("--volume-path", type=str, required=True)
+def test(
+    host: str,
+    username: str,
+    password: str,
+    catalog: str,
+    volume: str,
+    volume_path: str,
+):
+    client = WorkspaceClient(host=host, username=username, password=password)
+    files = list(
+        client.files.list_directory_contents(_get_volume_path(catalog, volume, volume_path))
+    )
+
+    assert len(files) == 1
+
+    resp = client.files.download(files[0].path)
+    data = json.loads(resp.contents.read())
+
+    # Are these assertions too opinionated?
+    # We don't need tests failing when we change how we process the data
```

Comment:
If we are just running this every so often, this is great. 

Let's come to consensus on if we want to run databricks-volumes.sh on CI. If so, then maybe.. but I think it's better to be more opinionated.

Think of the standard CI expected-structured-output. They are super detailed/opinionated.

Reply:
Should I be more opinionated here and have tests that do exact matches for output?

Reply:
I don't think so. We do super opinionated for source connectors. For destination its usually less opinionated. And then on vector destinations we do a vector search.

What you have is fine. I think for destination we just need to test that its a real file. If there are errors in partitioning,chunking,embedding those should be exposed by source tests anyway.

Reply:
Cool! Removed comment :)